core/connection/connect_options.go

```diff
@@ -17,8 +17,24 @@
 
 package connection
 
+import (
+	"encoding/json"
+
+	"github.com/mysteriumnetwork/node/identity"
+	"github.com/mysteriumnetwork/node/session"
+)
+
 // ConnectOptions holds connect options
 type ConnectOptions struct {
 	// kill switch option restricting communication only through VPN
 	DisableKillSwitch bool
 }
+
+// Params represents the params we need to ensure a successful connection
+type Params struct {
```

Comment:
Struct looks like a duplicate of ConnectOptions.

A. Suggest to merge to `ConnectOptions`

B. Maybe:
- `Params` -> `ConnectOptions`
- `ConnectOptions` -> `ConnectionParams` -> represents plugin specific params